File: src/main/java/net/rainmore/platform/web/Application.java
package net.rainmore.platform.web;


import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

@Configuration
@EnableAutoConfiguration
@ComponentScan(basePackages = {"net.rainmore.platform.web"})
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: src/main/scala/au/com/rainmore/rest/Application.scala
package au.com.rainmore.rest

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.context.annotation.{ComponentScan, Configuration}


object Application extends App {
    SpringApplication.run(classOf[Application], args: _*)
}

@Configuration
@EnableAutoConfiguration
@ComponentScan(value = Array("au.com.rainmore.rest"))
class Application {}
